frontend/tests/common/constants.ts

```diff
@@ -7,7 +7,25 @@ const TEST_ENV_TO_TEST_URL = {
   local: "http://localhost:8000",
   localProd: "http://localhost:9000",
   prod: "https://cellxgene.cziscience.com",
+  rdev: process.env.RDEV_LINK || "",
   staging: "https://cellxgene.staging.single-cell.czi.technology",
 };
 
+const TEST_ENV_TO_TEST_EMAIL = (env: TEST_ENV) => {
+  switch (env) {
+    case "dev":
+    case "rdev":
+      return "cellxgene-smoke-test+dev@chanzuckerberg.com";
+    case "staging":
+      return "cellxgene-smoke-test+staging@chanzuckerberg.com";
+    case "prod":
+      return "cellxgene-smoke-test+prod@chanzuckerberg.com";
```

Comment:
If this test will run on prod we should avoid creating test collections. We don't want to end up with a bunch of junk collections if this test fails.

Reply:
that's a good concern! Maybe we can only create a new collection only if there's less than 2 collections, otherwise some other tests might rely on the account having at least 1 collection to run their tests?

Reply:
For now, I've just disabled the tests on prod